src/Infrastructure/Data/Configurations/ChatConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using CoduTeam.Domain.Entities;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CoduTeam.Infrastructure.Data.Configurations;
+
+public class ChatConfiguration: IEntityTypeConfiguration<Chat>
+{
+    public void Configure(EntityTypeBuilder<Chat> builder)
+    {
+        builder.HasKey(i => i.Id);
+        builder.Property(i => i.Id)
+            .ValueGeneratedOnAdd();
+        
+        builder.HasMany<Message>(a => a.Messages)
```

Comment:
It's deal of habit but better to create this relations in child tables, cause we have navigation field there